contributed/DIA_Run2_intro.ipynb

```diff
@@ -0,0 +1,2123 @@
WsN8RyQvfrcSmqlhSkhgRzn9uvoaMuHNHCPuyV99if0FhrfecmsZ9HdK4/4JAqwR+dNNk1N//Tam3tstvZpeOrHHrPHdFywk01P2tGouUkl99/Q3z9uzBpVdWurDZuHHAAC6d1Lg2LWnyydGnOVyxFV160YQdgeCGzk/TKbxfUG0Euj+PVGzl4yNPYkgvReURfLR9DMj6Bk4A3197TbVsoSUtdpRuYFPRClShMiJ+Aj0bEXFsScnTGxbw/t7N2FUNn2UyJiWDf467mnAtOMGLt3Zt4NkNS/CYBhIIUzW6Rsfz6bTbcKqhMxVt5bnWosZPShlsXPp7wBwqjZ8QYiDwOjBVSlkQ4PMngFghhCalNPC7RrMrt2UBnYAsIYQGxACFAdpocaLDnGhCRcdosIBmn+T2IIvTcbSkhBnv/g+X7pc7y69w8dj8+WSVltT7bIVP56U1q/AaJl7D71bK0ku5673P+Ob+28+JIIOc0jKySuqfu8cw+HTbjlMav2OlZRhWgHUzCWuOZIWymyFHCMFfLruEyzN78cWOXWiK4Op+fRndqVOj2sl1lfHh3u/JKR/NwJQRpEWXEKnF0CfmIsLU5gk7Z0QM5NLUn/Ht8VdIjDSwaQY+3VZL5B5gUFqHasMnpeR/h19kZ9kmfJZ/UPJ9yTrGJF7ClUFK7L2zZwMf7NtaS+5w5fHD/GbNfJ6/4PQ5q2U+L3+uNHxVuE2Dg6VFfLJvG7f0HhJUP84mWjPas6eUcm/ly+nArsr3M4BP8Udv7gm0r5RSCiEWA9fhj/j8EfBF5ebZla9XVW5fJFvJt2tXVW4fMYRX16zDlNQygAK4achAEiLO/oT0lubl1Wtw63otd5fbMPjHqtXEV7r8qiiocOHWa0cDWlKSX17Bztx8+nZIPmP9bg1OZ9rjwpyBjR+QHBkR+g6FGCEE47t2ZXzXptVXXH38CHcu+ATDsvBZJp8fsNE5KpZPLp9CmBoaScB+sRfTJ2YCJXoumWoBv5zzLYbhryEvEEQ4bLxy9VXVnz9QsbOW4QPwSS8rTnzNmITJJDoCOc9q88bOdbjN2uuePstkzpGd/Hn0VBynmbltzD+GTVHw1FmGdJs6847saTd+IeZZIURv/KkOh/FHeoI/6jMBeLlylG5U+YGFEHOBu6WU2fjTIWYJIf4IbALeqNz/DeAdIcQ+/DO+G8/Q+QTkoQkXoAjB62vW46t86ETa7Tw84QJuHjqoNbt21rDu2LFaFbmrUBUFj2EQbjup+GI28GBXFEGJ2xNw29lGanQUGbEx7DtRWMuf79Q0rhl4apddjNPJ5J7d+Xbv/lpFkhUBPxkzsoV63DawpOTn331VS/7PZegcKC3kPzvW89NBY0N2LEWoxNnTuLx3Gt3jO/DfjZs4WlLChZ0zuGXQIKKdJ6O8d5RuqGX4qhDArrLNXOg4fQHeUj3wtS0Bt6HXM35us4L1hUvIcu2nQ1gGDrVvwNgEAcQ5mp+W1RZpNeMnpby2gffvBgJmX0opp9X4+wBQ726VUnqA60PUzWajCMFDEy7gwYvG4NZ1Iuz2c8L1diZJj4nmYFFRvfd9pklknSCHaKcDp6bVywUzLIuBHU8/gg7ElpzjfLF9J5aUXNGnN8PTOzapnVDy4ozLuendj9BNE49h4NQ0+qYkcefIoafd99mpl/ALcx6L9u2naqgggb0nCpjco/EVM4o9br45sA+XoTMhoytdYuMaf0JngAMlhZT56hsZr2nyxYGdITV+NclMSuLZSy9pcLtTCUdFxaT2tEsIFacSnOEZk9KZb47uwaoT3pASFkmMvXY6VZEvn5f2PIbP8qBLH1qJHU3YiLIPx2X4ag+oVI3bzsFZH7QrvJwxVEUJmSDz+cb9o0axLutYLYPmUFUu6tKFb0vKyS8tZ/RT/2Rc765MSXKQHhtNVnEpHsOoDIrQeGzyeCLsjXdr/XXpCv67fiNe01+j8ZPvt/ODgf357cUTQ3iGjadnUiJLH7ib+bv3crysnMFpHRjduVNQhivcbqNLXCw2Ta1eF5XAK6vWkBQRzvUD+wfdj8WHD3L/17MRCExp8eeVS7lr0DAeHdP2CsDaKwOgAm5TWu9ROCx+HIvyPq+sql6b/jHB5QI+OngCK48fwm3o6NJCEQKHovKnUVPrXROzj72JyyyrjnQ1pA9D+riidz5z96RQ4vUiBOiWxS+HXMTIlMatqZ4ttBu/NkiFz0eBy9WsfKNziZHp6Tx76SX8YdHi6qCXS3v2xJ3n5XhxGZaUlHl8fL11Dz0GduTNm6/hm937Wbh7H4mREdw6fBBD0huvVn+wsIg312+sZXTdusGHK7eyZsVBCkpcdEmJ4+fTL2R05pmvth5ut3H1gMZLcRmWxf82b6k2fFW4DYN/rV4btPFz6ToPfP0l7jqz7De3bmRi566MSKuXotuqZETF0ikqln3FJ2rdW2GajVsyW28JIt6ezA2d7uODo/+qpSxze5eHcarBxQR0jY7n6yvu5vUda9lwIotu0Qnc22cUmXH117h3l22pl+IBUGbtZOnVv2XLieOU+LwMS+pIjOPcFeFoN35tCN00eXrxYj7+fjuqonBfhxT2r1/P3cPPWOpLm+XKzEym9epFbnk50U4nx4vK+MHf36s1krekxJKSeZt3c/tFw7ltxOBmHXPR/gP1FEW0ClCKLLKkP9py59E8Zr46m+fvmc7YPmfeADYFj24ETBYHf8BQsCw9eqhWsFF1+4bBJ7t3tDnjB/DvSTO4Yd77uA0d0/I7CSd36s4NPYNXh2kJBseNJTN6CPvLt6MIlR6R/bApjfNUpIZH89vhpw+w14SGIevX81OEiiIUhiW3vd+tJWg3fm2Ivyxdxifbd/gDEUwTS0peWLGS5IhIpvfJbO3utTqqopAWHQ3AnuMn0NST+VNSSCyb3wBuPnK80W3nu5ZxtHQWhiynQ/hUOkZdjVPT/MVuq1bGJNhLQNQZNHt0gxe/WHbWGL8Iu43EiHCOl9WvYtC/Q0r
```

Comment:
Add some more text explaining the steps you're going to do below.  Something like

```
1. Load DIAObject table and select an object
2. Get the diaSrc IDs from the DIAObject table for that object
3. Use a magic trick to get the dataIds that allow us to load the diaSrc catalogs for each observation of the source.  These are the measurements from the detections on each image.  They are _not_ the forced photometry on all available images.
4. Load diaSrc values for each detection.
5. Calibrate the diaSrc instFlux measurements
6. Plot lightcurve
```

Reply:
Thank you for this, it really improves the introduction part.